package.json

```diff
@@ -42,7 +42,7 @@
     "postcss-value-parser": "^3.3.0",
     "stdin": "0.0.1",
     "stylelint": "^7.5.0",
-    "stylelint-order": "^0.3.0"
+    "stylelint-order": "0.4.x"
```

Comment:
Why not `^0.4.0` ? The author of stylelint-order said, he follows semantic versioning :) https://github.com/morishitter/stylefmt/issues/266#issuecomment-286573961